install_config/cluster_metrics.adoc

```diff
@@ -1031,6 +1034,118 @@ Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
 gathered from the `http://${POD_IP}:7575/metrics` endpoint.
 ====
 
+[[openshift-prometheus-kubernetes-metrics]]
+=== {product-title} Metrics via Prometheus
+
+The state of a system can be gauged by the metrics that it emits. This section
+describes current and proposed metrics that identify the health of the storage subsystem and
+cluster.
+
+[[k8s-current-metrics]]
+==== Current Metrics
+
+This section describes the metrics currently emitted from Kubernetes’s storage subsystem.
+
+*Cloud Provider API Call Metrics*
+
+This metric reports the time and count of success and failures of all
+cloudprovider API calls. These metrics include `aws_attach_time` and
+`aws_detach_time`.  The type of emitted metrics is “histogram," and hence,
+Prometheus also generates sum, count, and bucket metrics for these metrics.
+
+.Example summary of cloudprovider metrics from GCE:
+----
+cloudprovider_gce_api_request_duration_seconds { request = "instance_list"}
+cloudprovider_gce_api_request_duration_seconds { request = "disk_insert"}
+cloudprovider_gce_api_request_duration_seconds { request = "disk_delete"}
+cloudprovider_gce_api_request_duration_seconds { request = "attach_disk"}
+cloudprovider_gce_api_request_duration_seconds { request = "detach_disk"}
+cloudprovider_gce_api_request_duration_seconds { request = "list_disk"}
+----
+
+.Example summary of cloudprovider metrics from AWS:
+----
+cloudprovider_aws_api_request_duration_seconds { request = "attach_volume"}
+cloudprovider_aws_api_request_duration_seconds { request = "detach_volume"}
+cloudprovider_aws_api_request_duration_seconds { request = "create_tags"}
+cloudprovider_aws_api_request_duration_seconds { request = "create_volume"}
+cloudprovider_aws_api_request_duration_seconds { request = "delete_volume"}
+cloudprovider_aws_api_request_duration_seconds { request = "describe_instance"}
+cloudprovider_aws_api_request_duration_seconds { request = "describe_volume"}
+----
+
+See
+link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cloud-provider/cloudprovider-storage-metrics.md[Cloud
+Provider (specifically GCE and AWS) metrics for Storage API calls] for more
+information.
+
+*Volume Operation Metrics*
+
+These metrics report time taken by a storage operation once started. These
+metrics keep track of operation time at the plug-in level, but do not include
+time taken by `goroutine` to run or operation to be picked up from the internal
+queue. These metrics are of type histogram.
```

Comment:
Histogram is in ""s above. Should this one too?

Reply:
Fixed.